app/views/shared/_post_rendering.html.erb

```diff
@@ -0,0 +1,29 @@
+<table>
+  <thead>
+    <tr>
+      <th>Grumblr</th>
+      <th>Content</th>
+      <th>Grass tags</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+    <% @posts.each do |post| %>
+      <div class="post-<%= post.id %>" >
+        <tr>
+          <td><%= link_to @users.where(id: post.user_id)[0].name, @users.where(id: post.user_id)[0] %>:</td>
```

Comment:
I don't even want to know what this line would have had to look like had I not put `:user_id` on Posts